ui/admin/tests/unit/abilities/credential-test.js

```diff
@@ -4,87 +4,74 @@ import { setupTest } from 'ember-qunit';
 module('Unit | Abilities | credential', function (hooks) {
   setupTest(hooks);
 
-  test('canRead', function (assert) {
+  let features;
+
+  hooks.beforeEach(function () {
+    // Enable json-credentials feature by default
+    features = this.owner.lookup('service:features');
+    features.enable('json-credentials');
```

Comment:
Do we need to specify this here? `json-credentials` are on by default in the testing environment? Although I am all for being explicit so this is non-blocking.

Reply:
Ideally each test is self-contained and explicitly manages its own feature flags.  My reasons for this:

- If we explicitly set feature flags per test file, we don't need to rely on a global default for tests.  Global defaults become brittle since any changes will almost certainly break existing tests.
- For readability, it's nice to see up front which flags the tests rely on.

Open to discussion.

Reply:
Discussed this with the team. All in favor of explicitly managing feature flags per test file.